packages/app/src/runner/selector-playground/SelectorPlaygroundTooltip.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <Tooltip
+    :triggers="[]"
+    :hide-triggers="[]"
+    :distance="8"
+    hide-arrow
+    :auto-hide="false"
+    :shown="shown"
+    handle-resize
+  >
+    <div
+      :key="'popover-container'"
+      @mouseenter="mouseEnter"
+      @mouseleave="mouseLeave"
+      @click="click"
+    >
+      <slot />
+    </div>
+    <template #popper>
+      <div
+        class="whitespace-nowrap"
+        data-cy="selector-playground-tooltip"
+      >
+        {{ textToShow }}
+      </div>
+    </template>
+  </Tooltip>
+</template>
+
+<script lang="ts" setup>
+import { ref } from 'vue'
+import Tooltip from '@packages/frontend-shared/src/components/Tooltip.vue'
+
+const props = defineProps<{
+  hoverText: string
+  clickText?: string
+}>()
+
+const shown = ref(false)
+const textToShow = ref(props.hoverText)
+
+function mouseEnter () {
+  textToShow.value = props.hoverText
+  shown.value = true
+}
+
+async function mouseLeave () {
```

Comment:
No need for async here with your changes 😃

Reply:
Addressed in [aa185df](https://github.com/cypress-io/cypress/pull/21334/commits/aa185df92647260a331d523fe57b31cd4850629e)